schemas/com.android.installreferrer.api/referrer_details/jsonschema/1-0-0

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Represents an install referrer details for Android apps installed from the Play Store (see https://developer.android.com/reference/com/android/installreferrer/api/ReferrerDetails)",
+	"self": {
+		"vendor": "com.android.installreferrer.api",
+		"name": "referrer_details",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"installReferrer": {
+			"type": ["string", "null"],
+			"maxLength": 2048,
```

Comment:
URLs can be unbelievably long sometimes. Wonder if its worth making this bigger?

Worth going up to 4096? Matches the atomic page_url?
https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0#L124-L127

Reply:
Makes sense, changed to 4096!